pkg/processor/runtime/python/py/_nuclio_wrapper.py

```diff
@@ -247,6 +251,13 @@ def _on_termination_signal(self, signal_number, frame):
             # after the current event is handled
             self._is_termination_needed = True
 
+    def _on_continue_signal(self, signal_number, frame):
+        self._logger.debug_with('Received continue signal',
+                                signal=signal.Signals(signal_number).name)
+
+        # set this flag to False, so continue normal event processing flow
+        self.discard_events = False
```

Comment:
Where is this set to `True`?
Should be inside `_on_drain_signal`

Reply:
@TomerShor Yes, it should be. This code hasn't been tested or polished yet. I decided to share my work since parallel work was ongoing with the same code part. Will add!